src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robo = {};
+
+  for (const key in robot) {
+    if (robo[robot[key]]) {
```

Comment:
```suggestion
    if (robo[robot[key]]) {
```

move robot[key] to separate variable